src/frontend/src/lib/components/transactions/AllTransactionsList.svelte

```diff
@@ -9,9 +9,13 @@
 	import { mapAllTransactionsUi } from '$lib/utils/transactions.utils';
 
 	let transactions: AllTransactionsUi;
+	// TODO: add ethTransactions to mapAllTransactionsUi
 	$: transactions = mapAllTransactionsUi({
 		tokens: $enabledTokens,
-		$btcTransactions: $btcTransactionsStore
+		$btcTransactions: $btcTransactionsStore,
+		$ethTransactions: {},
+		$ckEthMinterInfo: {},
+		$ethAddress: undefined
```

Comment:
undefined? not needed?

Reply:
it is just a placeholder, since the UI part will come in another PR, as discussed

Reply:
ok